api/lib/spree/api/testing_support/v2/platform_contexts.rb

```diff
@@ -50,12 +50,12 @@ def articleize
 
 def json_api_include_parameter(example = '')
   let(:include) { nil }
-  parameter name: :include, in: :query, type: :string, descripton: JSON_API_INCLUDES_DESCRIPTION, example: example
+  parameter name: :include, in: :query, type: :string, description: JSON_API_INCLUDES_DESCRIPTION, example: example
 end
 
 def json_api_filter_parameter(example = '')
   let(:filter) { nil }
-  parameter name: :filter, in: :query, type: :string, descripton: JSON_API_FILTER_DESCRIPTION, example: example
+  parameter name: :filter, in: :query, type: :string, description: JSON_API_FILTER_DESCRIPTION, example: example
```

Comment:
I'm not sure if these two changes somehow might make the tests fail.

Reply:
the description is an optional parameter so no worries, it didn't fail with the typo ;)